docs/docs/api/Tasks/fetchTaskByDay.md

```diff
@@ -23,17 +23,11 @@ This is a **protected route**, a **valid JWT is required** in the header field
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
-#### Body
-
-```json
-{}
-```
-
 #### Parameters
 
 ```json
 {
-  "day": "yyyy-MM-dd'T'HH:mm:ss" // ISO formatted START date
+  "day": "yyyymmdd"
```

Comment:
For this portion, just to keep it consistent with the other file, I would just remove the ```json``` and have it as 
`"day" : "yyymmdd"`. Similar to the way how, it's done for Fetch Single Task

![image](https://user-images.githubusercontent.com/24628243/161081622-9cd88af4-b284-4350-b9dc-0f47cfe1b7a0.png)